### PR TITLE
Feature/simple variants

### PR DESCRIPTION
**Pull request by adrien-louis-r**
Improve the configurator appearance and add axis information.

### CODE REVIEW

File: plugins/gatsby-source-sylius/createSchemaCustomization.js [hunk 1]
```diff
@@ -15,9 +15,10 @@ module.exports = ({ actions }, options) => {
     }
 
     type Variant {
-      name: String!,
+      name: String,
```
Comment (gkueny) [inline plugins/gatsby-source-sylius/createSchemaCustomization.js]: A variant can not have a name ? 😱

File: src/components/Configurator/Configurator.js [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+            />
+          </OptionLeft>
+        </OptionItem>
+      ))}
+    </OptionList>
+  );
+};
+
+Configurator.propTypes = {
+  variants: PropTypes.array.isRequired,
```
Comment (gkueny) [inline src/components/Configurator/Configurator.js]: Variants propTypes can be more exhaustive :  
```suggestion
  variants: PropTypes.arrayOf(PropTypes.shape({
    code: PropTypes.string.isRequired,
    price: Proptypes.shape({
      current: PropTypes.string.isRequired,
      currency: PropTypes.string.isRequired
    }).isRequired
  })).isRequired,
```

(I'm no really sure of my proposition code)

File: plugins/gatsby-source-sylius/createSchemaCustomization.js (hunk 1 above)
Comment (adrien-louis-r) [reply plugins/gatsby-source-sylius/createSchemaCustomization.js]: Yes, it is not required in Sylius backoffice.

File: src/components/Configurator/Configurator.js (hunk 2 above)
Comment (adrien-louis-r) [reply src/components/Configurator/Configurator.js]: Thanks, I'll update it on my side to test the shape validity

**Pull request merged**